### PR TITLE
dumpEntry should ignore non-exist inode

### PR DESCRIPTION
**Pull request by polyrabbit**

### CODE REVIEW

**Comment by davies** [comment]
Right now, we dump it as empty one (could be deleted or corrupted)

File: pkg/meta/tkv.go [hunk 1]
```diff
@@ -2744,6 +2744,7 @@ func (m *kvMeta) dumpEntry(inode Ino, e *DumpedEntry, showProgress func(totalInc
 		a := tx.get(m.inodeKey(inode))
 		if a == nil {
 			logger.Warnf("inode %d not found", inode)
+			return nil
```
Comment (SandyXSD) [inline pkg/meta/tkv.go:2747]: If this is a directory, we still want all its children.

Comment (polyrabbit) [reply pkg/meta/tkv.go:2747]: OK, so a directory will also trigger a warning message? I see it a lot in the log

Comment (davies) [reply pkg/meta/tkv.go:2747]: so we can improved the case that file are delete after we saw the entry (during the dump)

**Comment by polyrabbit** [comment]
Closing as it's by design

**Pull request closed without merging**